bundler.js

```diff
@@ -148,8 +148,18 @@ export async function bundleInstall(gemfile, lockFile, platform, engine, rubyVer
     console.log(`Found cache for key: ${cachedKey}`)
   }
 
+  const installFlags = [
+    '--jobs', '4',
+  ]
+
+  // Use binary cache if available
+  const vendorCachePath = process.env['BUNDLE_CACHE_PATH'] || 'vendor/cache'
```

Comment:
This mirror's Bundler own configuration (as documented [here](https://bundler.io/v2.2/man/bundle-config.1.html#LIST-OF-AVAILABLE-KEYS))